examples/amm/src/lib.rs

```diff
@@ -31,7 +31,7 @@ pub enum Operation {
     /// Given an input token idx (can be 0 or 1), and an input amount,
     /// Swap that token amount for an amount of the other token,
     /// calculated based on the current AMM ratio
-    /// Owner here is the user executing the Swap
+    /// MultiAddress here is the user executing the Swap
```

Comment:
```suggestion
    /// `owner` here is the user executing the Swap
```
or
```suggestion
    /// The [`MultiAddress`] here is the user executing the Swap
```

(Also below.)